lib/util/sdp/index.js

```diff
@@ -251,6 +289,84 @@ function setSimulcast(sdp, sdpFormat, trackIdsToAttributes) {
   })).concat('').join('\r\n');
 }
 
+/**
+ * Filter local codecs based on the remote unified plan SDP.
+ * @param {string} localSdp
+ * @param {string} remoteSdp
+ * @returns {string} - Updated local SDP
+ */
+function unifiedPlanFilterLocalCodecs(localSdp, remoteSdp) {
```

Comment:
lot of munging going in here that relies on sdp formats - would be useful to put in inside try catch block and log an error - this would let you know if any of the assumptions about format were wrong and thus led to crash.